<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>sign_in_widget</class>
 <widget class="QMainWindow" name="sign_in_widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>584</width>
    <height>361</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QT</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="w1">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>70</y>
      <width>194</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="w2">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>120</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="sign_in_btn">
    <property name="geometry">
     <rect>
      <x>292</x>
      <y>231</y>
      <width>80</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>sign in</string>
    </property>
   </widget>
   <widget class="QPushButton" name="regis_btn">
    <property name="geometry">
     <rect>
      <x>193</x>
      <y>231</y>
      <width>80</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>register</string>
    </property>
   </widget>
   <widget class="QComboBox" name="ident_com">
    <property name="geometry">
     <rect>
      <x>133</x>
      <y>180</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>订单者</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>商家</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>管理员</string>
     </property>
    </item>
   </widget>
   <widget class="QLineEdit" name="account_line">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>80</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="account_label">
    <property name="geometry">
     <rect>
      <x>175</x>
      <y>80</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>账号:</string>
    </property>
   </widget>
   <widget class="QLabel" name="key_label">
    <property name="geometry">
     <rect>
      <x>174</x>
      <y>130</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>密码:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="key_line">
    <property name="geometry">
     <rect>
      <x>219</x>
      <y>130</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>584</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
